lib/Tvm.js

```diff
@@ -36,6 +36,14 @@ class Tvm {
       // owAuth is redundant here as we already check it but it cannot hurt to check twice
       owAuth: joi.string().required()
     }
+
+    const tempSchema = {
```

Comment:
that schema is specific to azure blob signature, it should be set elsewhere, the class is global to all endpoints